src/test/java/com/baidu/hugegraph/BaseClientTest.java

```diff
@@ -30,18 +31,24 @@
 
 public class BaseClientTest {
 
-    protected static String BASE_URL = "http://127.0.0.1:8080";
-    protected static String GRAPH = "hugegraph";
+    protected static final String BASE_URL = "http://127.0.0.1:8080";
+    protected static final String GRAPH = "hugegraph";
+    protected static final String USERNAME = "admin";
+    protected static final String PASSWORD = "pa";
+    protected static final int TIMEOUT = 10;
 
     private static HugeClient client;
 
     protected static HugeClient open() {
-        return new HugeClient(BASE_URL, GRAPH);
+        client = new HugeClient(BASE_URL, GRAPH, USERNAME, PASSWORD);
```

Comment:
just return `new HugeCliennt....`

Reply:
to set static field that will be used by other methods like clear()